src/strings/en-GB.json

```diff
@@ -82,6 +82,7 @@
   "LabelUrl": "URL:",
   "LabelZipCode": "Post Code:",
   "LabelffmpegPathHelp": "The path to the ffmpeg application file or folder containing ffmpeg.",
+  "LeaveBlankToNotSetAPassword": "This field can be left blank.",
```

Comment:
I'd actually rather this be something like: `Optional - leave blank to set no password.`, both descriptive and functional.